src/poliastro/twobody/orbit.py

```diff
@@ -368,12 +369,23 @@ def from_equinoctial(
 
     @classmethod
     def from_body_ephem(cls, body, epoch=None):
-        """Return osculating `Orbit` of a body at a given time.
 
-        """
+        """Return osculating `Orbit` of a body at a given time."""
+
         # TODO: https://github.com/poliastro/poliastro/issues/445
+
+        if (
+            body.name == "Pluto"
+            and body.name.lower() not in solar_system_ephemeris.bodies
+        ):
+            raise KeyError(
+                """Default Ephemeris selected. To change it, please do
```

Comment:
I would rather do: 

```suggestion
                """These bodies can not be found in the "builtin" ephemeris. Please change the ephemeris.
                >>> from astropy.coordinates import solar_system_ephemeris
                >>> solar_system_ephemeris.set('jpl')"""

```

Reply:
And I am not sure if raising `KeyError` is the correct thing to do!

Reply:
i was thinking to set it as Not-Implemented Error, but have to change test cases accordingt to it.

should I do that?
and change test cases also?